htdocs/GetStatic.php

```diff
@@ -33,7 +33,13 @@
 // Ensures the user is logged in, and parses the config file.
 require_once "NDB_Client.class.inc";
 $client = new NDB_Client();
-if ($client->initialize("../project/config.xml") == false) {
+
+$configFile = getenv('LORIS_DB_CONFIG');
```

Comment:
Ah yes, thank you. I wrongly assumed that one of the default parameters along the way would override my null. I'll update this.